tuxemon/core/monster.py

```diff
@@ -229,8 +229,13 @@ def __init__(self, save_data=None):
 
         self.weight = 0
 
-        # the multiplier for checks when a monster ball is thrown.
-        self.catch_rate = 1
+        # The multiplier for checks when a monster ball is thrown this should be a value betwen 0-255 meaning that
+        # 0 is 0% capture rate and 255 has a very good chance of capture. This numbers are based on the capture system calculations.
+        # This is based on the pokemon calculation and can be found at https://bulbapedia.bulbagarden.net/wiki/List_of_Pok%C3%A9mon_by_catch_rate
+        self.catch_rate = 125
```

Comment:
Perhaps the default value for this could be in the game config?

Reply:
Yes of course. If you can, send me the location of the file and while assign the default value there

Reply:
/tuxemon/core/config.py

You'll need to update both the __init__ and the get_defaults()

Reply:
I suggest putting it in the gameplay section.

Reply:
Still wanna see this go to the config.

Reply:
I made the changes to config.py

Reply:
Great, but you also need to make the code *use* that config value :P

Reply:
Yeah, sorry my bad XD.  Is <code>self.catch_rate = TuxemonConfig.default_monster_catch_rate</code> ok? If there is another better way to do it please inform me,

Reply:
@xirsoi I have implemented the change.